README.md

```diff
@@ -79,6 +79,56 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
```

Comment:
This is the only element in the numbered list that does not follow the general form of:

1. Short description. Longer description that fills in what actions will need to be or will be taken.

Perhaps a simple "Create a project. Make an empty folder and switch..." for consistency?